packages/framework/pr_tools/PullRequestLinuxDriver.sh

```diff
@@ -4,37 +4,31 @@ SCRIPTPATH=$(dirname $SCRIPTFILE)
 source ${SCRIPTPATH:?}/common.bash
 # set -x  # echo commands
 
+# Fetch arguments
+on_weaver=$(echo "$@" | grep '\-\-on_weaver' &> /dev/null && echo "1")
+on_ats2=$(echo "$@" | grep '\-\-on_ats2' &> /dev/null && echo "1")
 
 
 # Load the right version of Git / Python based on a regex
 # match to the Jenkins job name.
 function bootstrap_modules() {
     print_banner "Bootstrap environment modules start"
+    message_std "PRDriver> " "Job is $JOB_BASE_NAME"
 
-    cuda_regex=".*(_cuda_).*"
-    ride_regex=".*(ride).*"
     vortex_regex=".*(vortex).*"
-    if [[ ${JOB_BASE_NAME:?} =~ ${cuda_regex} ]]; then
-        if [[ ${NODE_NAME:?} =~ ${ride_regex} ]]; then
-            message_std "PRDriver> " "Job is CUDA"
-            execute_command_checked "module unload git"
-            execute_command_checked "module unload python"
-            execute_command_checked "module load git/2.10.1"
-            execute_command_checked "module load python/3.7.3"
-            get_python_packages pip3
-            envvar_set_or_create PYTHON_EXE python3
-        elif [[ ${NODE_NAME:?} =~ ${vortex_regex} ]]; then
-            echo -e "Job is CUDA node is vortex"
-            execute_command_checked "module load git/2.20.0"
-            execute_command_checked "module load python/3.7.2"
-            get_python_packages pip3
-            envvar_set_or_create PYTHON_EXE python3
-        else
-            message_std "PRDriver> " "ERROR: Unable to find matching environment for CUDA job not on Ride."
-            exit -1
-        fi
+    if [[ ${NODE_NAME:?} =~ ${vortex_regex} || ${on_ats2} == "1" ]]; then
```

Comment:
Does ats2 not use the SEMS NFS?

Reply:
@fryeguy52 do you know?

Reply:
That's the only thing that catches my eye, but I don't really know anything about ats2. I just noticed it would not see the source /projects/sems/modulefiles/utils/sems-archive-modules-init.sh line in the else block. (soon to be non-archive) I only know of ride, vortex, and now weaver to do that, but ats2 could just as easily be in that same situation.

Looking a bit deeper into it, I'm seeing ats2 builds being cuda or using compiler versions that aren't sems or sems-archive, so they must use some other source. So it should be fine. Just wanted to check.